docs/reference/glossary-of-terms.md

```diff
@@ -0,0 +1,215 @@
+# Glossary of Terms
+
+This document is a list of terms that you might encounter in the ontology world. It is not an exhaustive list and will continue grow and evolve. Please do create a ticket if there is a term you find missing here, or a term that you encounter that you do not understand, and we will do our best to add them in here. This list is not arranged in any particular order, please use the search function to find needed term.
```

Comment:
Minor edits:
This document is a list of terms that you might encounter in the ontology world. It is not an exhaustive list and will continue to evolve. Please create a ticket if there is a term you find missing or a term you encounter that you do not understand, and we will do our best to add them. This list is not arranged in any particular order. Please use the search function to find needed term.

Reply:
> This list is not arranged in any particular order

Consider alphabetising.

Reply:
I can alphabetize this list. I'll do so on this branch.

Reply:
I revised the text per @bvarner-ebi's suggestion